test/jdk/java/net/httpclient/CancelRequestTest.java

```diff
@@ -377,6 +378,13 @@ public void testGetSendAsync(String uri, boolean sameClient, boolean mayInterrup
             assertEquals(cf2.isDone(), true);
             assertEquals(cf2.isCancelled(), false);
             assertEquals(latch.getCount(), 0);
+
+            var error = TRACKER.check(1,
+                    (t) -> t.getOutstandingOperations() > 0 || t.getOutstandingSubscribers() > 0,
+                    "subscribers for testInputStream(%s)\n\t step [%s]".formatted(req.uri(), i),
```

Comment:
Hello Daniel,
Is this a copy/paste error - the reference to `testInputStream` method name? Same with other places that have changed in this file.

Reply:
Gah! Yes - thanks for noticing.